[2021-03-04 22:51:13,943] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: clean_data.load_data 2021-03-04T22:51:08.529824+00:00 [queued]>
[2021-03-04 22:51:13,964] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: clean_data.load_data 2021-03-04T22:51:08.529824+00:00 [queued]>
[2021-03-04 22:51:13,964] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-03-04 22:51:13,964] {taskinstance.py:881} INFO - Starting attempt 1 of 2
[2021-03-04 22:51:13,965] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-03-04 22:51:13,994] {taskinstance.py:901} INFO - Executing <Task(PythonOperator): load_data> on 2021-03-04T22:51:08.529824+00:00
[2021-03-04 22:51:13,996] {standard_task_runner.py:54} INFO - Started process 521 to run task
[2021-03-04 22:51:14,021] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'clean_data', 'load_data', '2021-03-04T22:51:08.529824+00:00', '--job_id', '36', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/data_clean.py', '--cfg_path', '/tmp/tmp7yedrhke']
[2021-03-04 22:51:14,025] {standard_task_runner.py:78} INFO - Job 36: Subtask load_data
[2021-03-04 22:51:14,076] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: clean_data.load_data 2021-03-04T22:51:08.529824+00:00 [running]> 6623e80cd770
[2021-03-04 22:51:14,101] {data_clean.py:65} INFO - ****** NOAH starting load data
[2021-03-04 22:51:14,132] {logging_mixin.py:112} WARNING - /home/airflow/.local/lib/python3.6/site-packages/pandas/core/generic.py:2612: UserWarning: The spaces in these column names will not be changed. In pandas versions < 0.14, spaces were converted to underscores.
  method=method,
[2021-03-04 22:51:14,461] {data_clean.py:77} INFO - ****** NOAH 2 ending load data
[2021-03-04 22:51:14,463] {python_operator.py:114} INFO - Done. Returned value was: None
[2021-03-04 22:51:14,471] {taskinstance.py:1070} INFO - Marking task as SUCCESS.dag_id=clean_data, task_id=load_data, execution_date=20210304T225108, start_date=20210304T225113, end_date=20210304T225114
[2021-03-04 22:51:18,934] {local_task_job.py:102} INFO - Task exited with return code 0
